lib/FieldType/Mapper/IntegerFormMapper.php

```diff
@@ -61,7 +82,53 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
     }
 
     /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $validatorConfiguration = $fieldDefinition->getValidatorConfiguration();
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
+        $attributes = [];
+
+        if (null !== $validatorConfiguration['IntegerValueValidator']['minIntegerValue']) {
+            $attributes['min'] = $validatorConfiguration['IntegerValueValidator']['minIntegerValue'];
+        }
+
+        if (null !== $validatorConfiguration['IntegerValueValidator']['maxIntegerValue']) {
+            $attributes['max'] = $validatorConfiguration['IntegerValueValidator']['maxIntegerValue'];
+        }
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        IntegerType::class,
+                        [
+                            'required' => $fieldDefinition->isRequired,
+                            'label' => $label,
+                            'attr' => $attributes,
```

Comment:
@dpobel Isn't it the default value when `step` attribute is not provided? I can enforce it though, no problem.